### ISSUE TITLE
 List alternatives on readme

### ISSUE THREAD

**Issue by eloyesp** [sriharshaarangi/BatteryChargeLimit#195]
I found that listing alternative solutions on the readme is a very good practice, as it helps users make better choices (as that list usually explains differences between projects) and also helps developers join forces.

I've already mentioned this on https://github.com/MatteCarra/AccA/issues/132 as I found that AccA could be an alternative for this application.

**Comment by milux** [comment]
Absolutely, especially because of our lack of resources for further development during the last years.
Will definitely consider that.
If you want it to happen faster - please open a PR.
